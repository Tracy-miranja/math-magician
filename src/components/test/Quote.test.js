import React from 'react';

import renderer from 'react-test-renderer';
import Apps from '../Quote';

test('test the quote component', () => {
  const component = renderer.create(
    <Apps />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
