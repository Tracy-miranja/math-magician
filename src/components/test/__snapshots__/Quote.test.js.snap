// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the quote component 1`] = `
<section
  className="qqoutes"
>
  <div
    className="quotesbox"
  />
</section>
`;
